common.js

```diff
@@ -18,7 +19,54 @@ function isAuthenticated(req, res, next) {
     }
 }
 
+function getPendingPoints(userId){
+    var getRequestorPoints = `SELECT Transactions.pointCost AS pointCost
+    FROM Transactions
+    WHERE Transactions.requestorId = ? AND Transactions.statusId <> 8`;
+
+    var getSellerPoints = `SELECT Transactions.pointCost AS pointCost
+    FROM Transactions
+    INNER JOIN UserBooks on UserBooks.id = Transactions.userBookId
+    WHERE UserBooks.userId = ? AND Transactions.statusId <> 8`;
+
```

Comment:
Agree with Andrews points here where you will need to use callback or promises but callback does seem easier and it's how I've done some of my transactions now so just for consistency we could go with that.

Also agree with Andrew you could just do a sum in each select statement and then you can just sum the two result numbers together.

Your query is correct though. @jung8 Did you misread his queries? The '<>' means not equal so he's all transactions that are not closed which is more efficient than doing an equals or in statement on the other 7 statuses.

Reply:
The pending point column was the initial suggestion but since we decided to go this route and I have code that is already set up to use this I would ask that we don't suddenly change course as it will require me to do another refactor. This approach should be fine provided it's switched to use callback.

Reply:
@mkoeplin I think that if we select all statuses that are not equal to 8 then you select all statuses 1-7. This is not what we want for pending points because this includes received, rejected, canceled, and lost.